configure.ac

```diff
@@ -14,8 +17,8 @@ AC_ARG_VAR([GTEST_DIR], [directory containing Google Test])
 
 # Checks for programs.
 AC_PROG_AWK
-AC_PROG_CXX
-AC_PROG_CC
+AC_PROG_CXX([g++])
+AC_PROG_CC([gcc])
```

Comment:
Are these really necessary? These will limit detection of compilers... Maybe what would be better would be to move the flag-setting much higher, so that all the tests are done *with* the flags? Possibly even failing at the "detecting C++ compiler" step...